composer.json

```diff
@@ -24,7 +24,7 @@
     "require-dev": {
         "symfony-cmf/routing-bundle": "~1.2",
         "symfony-cmf/testing": "~1.1",
-        "sonata-project/doctrine-phpcr-admin-bundle": "~1.1",
+        "sonata-project/doctrine-phpcr-admin-bundle": "~2.0",
```

Comment:
lets point to the relevant dev branches here so that the PR can go green. we can then revert once the PRs on dependencies have been merged.